lib/management/paths.ts

```diff
@@ -44,6 +44,7 @@ export default {
     delete: '/v1/mgmt/tenant/delete',
     load: '/v1/mgmt/tenant',
     loadAll: '/v1/mgmt/tenant/all',
+    searchAll: '/v1/mgmt/tenant/search',
```

Comment:
why do we need the "all" term, why not just `search`

(Same for function name etc)

Reply:
This is how it is called in users as well

Reply:
don't you prefer to do it properly and not carry this further?